tests/supervisor/test_execution_online.py

```diff
@@ -0,0 +1,48 @@
+import asyncio
+
+import pytest
+from aleph_message.models import ItemHash
+
+from aleph.vm.conf import settings
+from aleph.vm.controllers.firecracker import AlephFirecrackerProgram
+from aleph.vm.models import VmExecution
+from aleph.vm.orchestrator import metrics
+from aleph.vm.storage import get_message
+
+
+@pytest.mark.asyncio
+async def test_create_execution():
+    """
+    Create a new VM execution and check that it starts properly.
+    """
```

Comment:
```suggestion
async def test_create_execution_from_network():
    """
    Create a new VM execution from the diagnostic VM and check that it starts properly.
    This will download the resources from the aleph.im network if they are not present locally.
    """
```

Reply:
Yes I like this suggestion, i'll make the changes!